packages/root-cms/ui/pages/TranslationsArbPage/TranslationsArbPage.tsx

```diff
@@ -148,3 +207,31 @@ function parseDocIds(docIdsText: string): string[] {
 function testValidDocId(docId: string): boolean {
   return !!docId && docId.includes('/') && docId.split('/').length === 2;
 }
+
+/**
+ * Builds a filename for the generated ARB file. The format is `<project>_<docId>_<timestamp>.json.arb`.
+ */
+function buildArbFilename(docIds: string[]) {
+  // Format date as `YYYYMMDDtHHMM`.
+  const now = new Date();
+  const pad = (n: number) => n.toString().padStart(2, '0');
+  const timestamp = `${now.getFullYear()}${pad(now.getMonth() + 1)}${pad(
+    now.getDate()
+  )}t${pad(now.getHours())}${pad(now.getMinutes())}`;
+  const projectName =
+    window.__ROOT_CTX.rootConfig.projectName ||
+    window.__ROOT_CTX.rootConfig.projectId;
+  const parts = [slugify(projectName)];
+  if (docIds.length === 1) {
+    parts.push(slugify(docIds[0]));
+  }
+  parts.push(timestamp);
+  return `${parts.join('_')}.json.arb`;
```

Comment:
is .json.arb a standard thing? if not, i'd prefer to keep it clean as just .arb.